episodes/3-documentation.md

```diff
@@ -674,9 +775,36 @@ autodoc2_packages = [
 ### Deploying documentation using Github pages
 
 - GitHub pages can be setup inside your GitHub repository
-- It automatically deploys your Sphinx-generated documentation. Or in other words it creates a website for you that renders your documentation.
+- It automatically deploys your MkDocs or Sphinx-generated documentation. Or in other words it creates a website for you that renders your documentation.
+
+::: challenge
+###### Exercise: Deploy documentation to GitHub Pages
+
+(If not done already) lets commit our changes to the main branch and push our changes to GitHub
+
+```bash
+git add mkdocs.yml
+git add docs/
+git add site/
+git commit -m "Add API documentation"
```

Comment:
This is not strictly limited to API documentation, so maybe:
`git commit -m "Add documentation"`